### ISSUE TITLE
Feature Request: Allow Ribbon to be applied other icons

### ISSUE THREAD

**Issue by ac87** [usefulness/easylauncher-gradle-plugin#78]
Really pleased to see this library picked up from where the last dev left off.

Whilst this works great for an app with a single entry point, we have an app with multiple launcher activities it would be good to be able to apply the ribbon to them all.

I believe you are currently limited to just the icon and round icon in the application tag of the manifest by: 

```
public static List<String> getLauncherIcons(File manifestFile) throws SAXException, IOException {
  GPathResult manifestXml = xmlSlurper.parse(manifestFile);
  GPathResult applicationNode = (GPathResult) manifestXml.getProperty("application");
  String icon = String.valueOf(applicationNode.getProperty("@android:icon"));
  String roundIcon = String.valueOf(applicationNode.getProperty("@android:roundIcon"));
...
```

But this doesn't check for activities or activity-alias'es with potentially a different icon and `android.intent.category.LAUNCHER`

A way to give the library a list of icon paths to ribbonize would be ideal, and probably a lot easier/cleaner than iterating the manifest.

**Comment by mateuszkwiecinski** [comment]
Hi @ac87 Thank you for reporting the issue! I see the usecase, and it is perfectly valid 👍 
And yeah, you're right - I wouldn't like to iterate over `activity-aliases` and automatically apply the ribbon - I see too many potential cases where it could break. The automatic discovery supposed to make the plugin application seameles for the simplest cases only. 

I'm wondering if a simple configuration would be sufficient:
```
easlylauncher {
  iconNames = ["@drawable/simple_icon", "@mipmap/another_icon"] // disables automatic icon discovery
}
```
or would you rather expect a more advanced, per icon, configuration? 

```
easylauncher { String iconName, BaseVariant buildVariant -> 
  if(iconName == "@drawable/simple_icon") {
    if(buildVariant.buildType == "debug") {
       filters = [...]
    }
  }
}
```

Also it would be great if you could provide a simple project which I could integrate as a test case to make sure it'll always work for your case. (Either here as one of [test projects](https://github.com/usefulness/easylauncher-gradle-plugin/tree/master/sample) or as a separe repository)

**Comment by ac87** [comment]
The first example would be sufficient for my use case. I just want to make it very clear to a user if they have a debug or beta version. 

I realized after I posted this you could have an app where the application icon is never shown in the launcher. I will provide an example project tomorrow.

**Comment by ac87** [comment]
Hi @mateuszkwiecinski 

Have added a sample here https://github.com/ac87/easylauncher-multiple-example it has 3 different sets of icons obviously currently it only decorates the one (two actually, includes round) defined in the application tag which isn't shown in the launcher.

I've added a mixture of round, legacy, and all types to test/demostrate everything.


Or at least it did. after testing various ways and including the fake keystore so I could include my enabled/disabled use case it now refuses to build for me with the following error:
```
Execution failed for task ':app:mergeDebugResources'... 
\easylauncher-multiple-example\app\build\generated\easylauncher\res\debug\drawable-v24-anydpi-v26: Error: Invalid resource directory name
```
Seems to be getting confused with the new icon type? Can raise this as a separate bug if you like

**Comment by mateuszkwiecinski** [comment]
Much appreciated! I'll have a look and integrate your project as a separate test cases and will fix the issue. Thanks!

**Comment by mateuszkwiecinski** [comment]
I have just released version `3.4.0` which should allow to provide custom set of icons using mentioned earlier api:
```groovy
easylauncher {
    iconNames = ["@drawable/icon_name"]
}
```

I integrated your project (thanks again for providing it!) into a test case and most of the variants are now covered with [screenshot tests](https://github.com/usefulness/easylauncher-gradle-plugin/tree/master/sample/example-activity-alias/screenshots). I'm saying "most of" as I'm unable to test what happens with `andorid:roundIcon` so it is not guaranteed to not work (it should though 🤷)

The error you have encountered:
>drawable-v24-anydpi-v26: Error: Invalid resource directory name

was caused by unexpected project setup where the foreground layer of the icon was placed under `drawable-v24` directory (adaptive icons are supported from api 26 onwards), but that wasn't anything _forbidden_ and the project should still compile. I added a dummy fallback that prevents such thing to happen in the future and the plugin should work even in such cases.

Please let me know if the new version addresses your initial problem 🤞

**Comment by ac87** [comment]
Tested with debug build and the multiple icons are all decorated as expected. Thank you very much.
